Pinta.Core/Classes/Document.cs

```diff
@@ -40,13 +40,20 @@ namespace Pinta.Core
 	// somewhat arbitrary.  In general:
 	// Document - Data about the image itself
 	// Workspace - Data about Pinta's state for the image
-	public class Document
+	public class Document : ObservableObject
 	{
 		private string filename;
 		private bool is_dirty;
 		private int layer_name_int = 2;
 		private int current_layer = -1;
 
+        // metadata
+        private string author;
```

Comment:
A lot of this code was based on the example of other code seen in Pinta. Also since I later made the dialog into a simple blocking/modal dialog it may not be necessary at all.